evalai/utils/submissions.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import sys
+
+from beautifultable import BeautifulTable
+from click import echo, style
+
+from evalai.utils.auth import get_request_header
+from evalai.utils.common import validate_token
+from evalai.utils.urls import URLS
+from evalai.utils.config import API_HOST_URL, EVALAI_ERROR_CODES
+
+
+def pretty_print_submission_table(submissions):
+    """
+    Funcion to print the submissions for a particular Challenge.
+    """
+    table = BeautifulTable()
+    attributes = ["participant_team_name", "method_name", "execution_time", "status", "submitted_at"]
```

Comment:
These attributes doesn't look good on the UI. 
Can you please look into it?
It should be ["Participant Team", "Method Name", "Execution Time(sec)", "Status", "Submitted At"]

Reply:
@RishabhJain2018 Changed!